langserve/server.py

```diff
@@ -554,6 +562,38 @@ def _route_name_with_config(name: str) -> str:
     InvokeResponse = create_invoke_response_model(model_namespace, output_type_)
     BatchResponse = create_batch_response_model(model_namespace, output_type_)
 
+    def _update_config_with_defaults(incomingConfig: RunnableConfig) -> RunnableConfig:
+        """Set up some baseline configuration for the underlying runnable."""
+
+        # Currently all defaults are non-overridable
+        overridable_default_config = RunnableConfig()
+        
+        metadata = {}
+        
+        is_hosted = os.environ.get("HOSTED_LANGSERVE_ENABLED", "false").lower() == "true"
+        if is_hosted:
+            hosted_metadata = {
+                "Commit SHA": os.environ.get("HOSTED_LANGSERVE_GIT_COMMIT", ""),
```

Comment:
Should the keys follow a particular format for rendering purposes / name collision purposes?

Reply:
I'm not sure, I started a thread internally to discuss the rendering questions. My worry on naming collision proof names is that they will be hard to read and thus hard to filter on